util/conditions/matcher.go

```diff
@@ -42,7 +42,7 @@ func (m matchConditions) Match(actual interface{}) (success bool, err error) {
 		elems = append(elems, MatchCondition(condition))
 	}
 
-	return gomega.ConsistOf(elems).Match(actual)
+	return gomega.ConsistOf(elems...).Match(actual)
```

Comment:
It's explicitly supported:
```go
//ConsistOf succeeds if actual contains precisely the elements passed into the matcher.  The ordering of the elements does not matter.
//By default ConsistOf() uses Equal() to match the elements, however custom matchers can be passed in instead.  Here are some examples:
//
//    Expect([]string{"Foo", "FooBar"}).Should(ConsistOf("FooBar", "Foo"))
//    Expect([]string{"Foo", "FooBar"}).Should(ConsistOf(ContainSubstring("Bar"), "Foo"))
//    Expect([]string{"Foo", "FooBar"}).Should(ConsistOf(ContainSubstring("Foo"), ContainSubstring("Foo")))
//
//Actual must be an array, slice or map.  For maps, ConsistOf matches against the map's values.
//
//You typically pass variadic arguments to ConsistOf (as in the examples above).  However, if you need to pass in a slice you can provided that it
//is the only element passed in to ConsistOf:
//
//    Expect([]string{"Foo", "FooBar"}).Should(ConsistOf([]string{"FooBar", "Foo"}))
//
```

Reply:
I guess that's something that you can only do if you have `... interface{}` as parameter and handle it accordingly.

So that's more like a false positive. But as both are valid I'm +1 to the change